### DISCUSSION TITLE
Allow custom backup rotation

### DISCUSSION THREAD

**Discussion by qk-santi** [nextcloud/all-in-one#2375] [Ideas] [answered]
### Is your feature request related to a problem? Please describe. 

Default backup rotation is "fine", but many people would have different environments.
Some people may want to backup changes for 2 years, while others don't care for anything older than 30 days.
It can greatly depend on the importance of the data stored.
Also, this can dictate a lot the backup storage space needs, so not being this configurable, may render some options unusuable.
Can't this be made configurable?

**Comment by szaimen** [comment, marked as answer]
This will be implemented with https://github.com/nextcloud/all-in-one/pull/2428

**Reply by szaimen** [reply]
This is now released with v6.0.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
